src/jit/importer.cpp

```diff
@@ -2903,14 +2915,12 @@ GenTreePtr      Compiler::impIntrinsic(CORINFO_CLASS_HANDLE     clsHnd,
     //
     // We disable the inlining of instrinsics for MinOpts.
     //
-    if  (opts.compDbgCode || opts.MinOpts())
+    if  ((opts.compDbgCode || opts.MinOpts()) && !(*pMustExpand))
```

Comment:
To avoid any changes impacting non-coreRT scenarios, i would suggest to add an assert that pMustExpand is false for non-coreRT scenarios.  Even better would be to take it into account only when compiling for CoreRT.